### DISCUSSION TITLE
Are code comments okay?

### DISCUSSION THREAD

**Discussion by cfuendev** [minbrowser/min#1837] [Q&A]
I'm implementing the multi-url bangs (https://github.com/minbrowser/min/discussions/1795) and I'm commenting some stuff along the way to keep in mind what everything does. Is that okay? Dunno if other contributors will find it helpful or rather annoying. 😟

**Comment by PalmerAL** [comment]
I think it's generally fine; it might be helpful depending on the comment. There's a document describing the overall architecture of Min: https://github.com/minbrowser/min/wiki/Architecture, you could consider updating that if what you're writing fits well there.
